Riot/Modules/Room/RoomViewController.m

```diff
@@ -5011,6 +5027,26 @@ - (void)declineRoomInvitation
     }
 }
 
+- (void)ignoreInviteSender
+{
+    [self startActivityIndicator];
+    [self.roomDataSource.room ignoreInviteSender:^{
+        [self stopActivityIndicator];
+        [self popToHomeViewController];
+
+    } failure:^(NSError *error) {
+        [self stopActivityIndicator];
+        MXLogDebug(@"[RoomVC] Failed to ignore inviter in room (%@)", self.roomDataSource.room.roomId);
+    }];
```

Comment:
Could be worth adding some `MXWeakify(self)`/`MXStrongifyAndReturnIfNil(self)`s in these?